### PR TITLE
Homework cleanup

### PR DESCRIPTION
**Pull request by BTBTravis**
We agreed in the call to have home work listed as follows.

### CODE REVIEW

File: homework/3_first_interactive_component.md [hunk 1]
```diff
@@ -16,4 +16,4 @@ For main project (current index.html):
 
 ### Submission:
 Don't forget to push to github!
-If you want to refresh knowlage about git workflow please follow these slides: <slides link here>
\ No newline at end of file
+If you want to refresh knowlage about git workflow please follow these slides: <slides link here>
```
Comment (aberezhk) [inline homework/3_first_interactive_component.md]: link shall be added: https://mrtim.github.io/js-munich-2020-spring/lessons/0_workflow/homework_workflow

Comment (aberezhk) [reply homework/3_first_interactive_component.md]: or https://mrtim.github.io/js-munich-2020-spring/lessons/2_git_time/

Comment (BTBTravis) [reply homework/3_first_interactive_component.md]: updated

File: homework/3_first_interactive_component.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Homework &mdash; First interactive component
+# Homework 3 -- First interactive component
```
Comment (MrTim) [inline homework/3_first_interactive_component.md:1]: What is the reason to use -- ?

Comment (BTBTravis) [reply homework/3_first_interactive_component.md:1]: consistent across all the assignments

Comment (BTBTravis) [reply homework/3_first_interactive_component.md:1]: no special reason I just edited the titles to all look the same.

Comment (MrTim) [reply homework/3_first_interactive_component.md:1]: Ok, I don't like those minuses and I would prefer to use either &mdash; or &ndash; everywhere. But I can leave with it ;-)

**Review by MrTim — APPROVED** [review]

**Pull request merged**